config/wrapper/no_run_tests.conf

```diff
@@ -65,20 +79,41 @@ tests =
 tests =
 [norun_sles12]
 tests =
+[norun_sles12.sp4]
+tests =
 [norun_sles12.sp4_NV]
 tests =
 [norun_sles12.sp4_pHyp]
 tests =
 [norun_sles12.sp4_kvm]
 tests =
+[norun_sles12.sp3]
```

Comment:
Suggestion: Better to move sp3 above sp4.

Reply:
okay